src/popup/popup.js

```diff
@@ -36,7 +36,14 @@ function submitLoginOnKeyEnter(event) {
 /**
  * Handle user's request to log in
  */
-async function handleOnClickLogin() {
+async function handleOnClickLogin(event) {
+    event.preventDefault();
+
+    let $form = document.getElementById("credential-input-forms");
```

Comment:
Can't this be `const`?

Reply:
Changed in 81501da5f1fde73623a60ff35b9753d684d0a6f3